packages/@aws-cdk/aws-pipes-targets-alpha/test/sns.test.ts

```diff
@@ -0,0 +1,81 @@
+import { InputTransformation, Pipe } from '@aws-cdk/aws-pipes-alpha';
+import { App, Stack } from 'aws-cdk-lib';
+import { Template } from 'aws-cdk-lib/assertions';
+import { Topic } from 'aws-cdk-lib/aws-sns';
+import { TestSource } from './test-classes';
+import { SnsTarget } from '../lib';
+
+describe('sns', () => {
+  it('should have only target arn', () => {
+    // ARRANGE
+    const app = new App();
+    const stack = new Stack(app, 'TestStack');
+    const topic = new Topic(stack, 'MyTopic', {});
+    const target = new SnsTarget(topic);
+
+    new Pipe(stack, 'MyPipe', {
+      source: new TestSource(),
+      target,
+    });
+
+    // ACT
+    const template = Template.fromStack(stack);
+
+    // ASSERT
+    template.hasResourceProperties('AWS::Pipes::Pipe', {
+      Target: {
+        Ref: 'MyTopic86869434',
+      },
+      TargetParameters: {},
+    });
+  });
+
+  it('should have input transformation', () => {
+    // ARRANGE
+    const app = new App();
+    const stack = new Stack(app, 'TestStack');
+    const topic = new Topic(stack, 'MyTopic', {});
+
+    const inputTransformation = InputTransformation.fromObject({
+      key: 'value',
+    });
+    const target = new SnsTarget(topic, {
+      inputTransformation,
+    });
+
+    new Pipe(stack, 'MyPipe', {
+      source: new TestSource(),
+      target,
+    });
+
+    // ACT
+    const template = Template.fromStack(stack);
+
+    // ASSERT
+    template.hasResourceProperties('AWS::Pipes::Pipe', {
+      TargetParameters: {
+        InputTemplate: '{"key":"value"}',
+      },
+    });
+  });
```

Comment:
See prior comment regarding the lack of coverage, I don't think it should be an issue given `InputTemplate` seems to be optional ([docs](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-properties-pipes-pipe-pipetargetparameters.html)):

```suggestion
  });
  
    it('should handle an empty parameter object', () => {
    // ARRANGE
    const app = new App();
    const stack = new Stack(app, 'TestStack');
    const topic = new Topic(stack, 'MyTopic', {});

    const inputTransformation = InputTransformation.fromObject({
      key: 'value',
    });
    const target = new SnsTarget(topic, {
      inputTransformation,
    });

    new Pipe(stack, 'MyPipe', {
      source: new TestSource(),
      target,
    });

    // ACT
    const template = Template.fromStack(stack);

    // ASSERT
    template.hasResourceProperties('AWS::Pipes::Pipe', {
      TargetParameters: {},
    });
  });
```

Reply:
I'm confused on this one. Isn't the "arrange" the same as the test above? Did you mean the following:
```
const target = new SnsTarget(topic, {});
```

Reply:
Yes I did, I updated the snippet sorry about that 👍